### PR TITLE
Add React v15 as peer dependency.

### PR DESCRIPTION
**Pull request by olafkarsten**
No problems running the code as is on react v15.

### CODE REVIEW

**Comment by albertogasparin** [comment]
+1, fixes #14

**Comment by albertogasparin** [comment]
Bump 😕

**Comment by antongrinenko** [comment]
Hi! 
Could you please release a new version with this pull request?

File: package.json [hunk 1]
```diff
@@ -41,8 +41,8 @@
   },
   "homepage": "https://github.com/WickyNilliams/react-simple-colorpicker",
   "peerDependencies": {
-    "react": "^0.14.0",
-    "react-dom": "^0.14.0"
+    "react": "^0.14.0 || ^15.1.0",
+    "react-dom": "^0.14.0 || 15.1.0"
```
Comment (dcousineau) [inline package.json]: I think you accidentally dropped a `^` on the react-dom 15.1 dependency.

I've also seen some projects use `"react": ">=0.14.0"` on their peer-dep

**Comment by WickyNilliams** [comment]
Fixed in 50e1ee4

**Comment by WickyNilliams** [comment]
(and published to npm as v1.4.0)

**Pull request closed without merging**